masterlist.yaml

```
@@ -11644,6 +11644,15 @@ plugins:
         util: '[SSEEdit v4.0.3h (Hotfix 1)](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 1
 
+  - name: 'Pirates Wield Cutlasses.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/31753/' ]
```

The next release of LOOT will likely expose locations in the UI, so it'd be beneficial to always include mod names for links going forward:

```suggestion
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/31753/'
        name: 'Pirates Wield Cutlasses'
```